atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/CachingTransaction.java

```diff
@@ -71,7 +71,11 @@ public Transaction delegate() {
     public SortedMap<byte[], RowResult<byte[]>> getRows(TableReference tableRef, Iterable<byte[]> rows,
                                                         ColumnSelection columnSelection) {
         if (Iterables.isEmpty(rows)) {
-            log.info("Attempted getRows on '{}' table and {} with empty rows argument", tableRef, columnSelection);
+            if (log.isTraceEnabled()) {
```

Comment:
Do we need the explicit conditions here? I'm not familiar with all of our logging libraries.

Reply:
`new Exception()` is expensive, so we want to make sure trace is enabled prior to allocating it, and filling in the call stack.

Reply:
Ah, I missed the exception in the log message. I don't like injecting the exception here. Is this just to force a stack trace that points to where the problem occurred?

Reply:
I'd actually propose to remove the `Attempted get*` logging completely at this point. If one calls with an empty rows/cells the empty result should be expected and I don't think this logging has ever actually been valuable.

Reply:
Agreed, I would be surprised if anyone used this.

Reply:
Works for me.